ci/test

```diff
@@ -30,12 +30,33 @@ all() {
   done
 }
 
-cucumber_authenticators() {
-  : DOC - Runs Cucumber Authenticator features
+cucumber_authenticators_config() {
+  : DOC - Runs Cucumber Authenticator configuration features
```

Comment:
I've noticed this `: DOC` thing a few places... what's it being used for?   Why not just use a comment?  Is automatic documentation being generated someplace?